### ISSUE TITLE
Building Examples using AnyCPU causes errors

### ISSUE THREAD

**Issue by ben51degrees** [51Degrees/device-detection-dotnet#2]
This only affects users that build directly from the solution file.
This is unlikely in practice as most people will just use the NuGet packages.

The problem is that the native DLLs are not produced in an AnyCPU flavour. Consequently, the build fails with a message like:
```
Could not copy the file "D:\Workspace\Pipeline\pipeline-dotnet\FiftyOne.DeviceDetection\FiftyOne.DeviceDetection.Hash.Engine.OnPremise.Native\AnyCPU\Debug\FiftyOne.DeviceDetection.Hash.Engine.OnPremise.Native.pdb" because it was not found.
```
This can be resolved by removing the AnyCPU configuration from the solution file.

**Comment by jwrosewell** [comment]
@ben51degrees Please can you go ahead and remove AnyCPU from the solution file in the next release.

**Comment by ben51degrees** [comment]
This change will be included in an upcomming release.

**Comment by ben51degrees** [comment]
Closed with [release 4.3.0](https://github.com/51Degrees/device-detection-dotnet/releases/tag/4.3.0%2B4). The specific commit removing the AnyCPU configuration is 2562e8eb30019450f37187c2073402ffed6286db.
